INSTALL.md

```diff
@@ -410,47 +410,28 @@ If you intend to run a server with translations, there are a few steps to follow
     This will enable you to compile and install the translations that are in
     the source repository.
     
-    If you want to get the latest translation files or partial work-in-progress
-    translations, or wish to work on translations yourself, you will need to
-    create a [Transifex](https://www.transifex.com/) account and install its
-    client software (`tx`):
+    If you want to work on translations yourself, you will need to create a
+    [MetaBrainz Weblate](https://translations.metabrainz.org/) account.
 
-        sudo apt-get install transifex-client
-
-    More information (and alternative ways to install the client) can be found
-    [here](https://docs.transifex.com/client/introduction/).
-
-    Next, [create an API token](https://www.transifex.com/user/settings/api/)
-    and use it to configure your credentials in
-    [`~/.transifexrc`](https://docs.transifex.com/client/client-configuration#-transifexrc).
-    
-    Finally, you will need to join the
-    [MetaBrainz Foundation organization](https://www.transifex.com/musicbrainz/public/)
-    on Transifex to get access to the translations. If you wish to work on
-    translations, you will also need to
-    [join a language team](https://www.transifex.com/musicbrainz/musicbrainz/dashboard/).
-    More information on how to get started can be found on
-    [the MusicBrainz site](https://musicbrainz.org/doc/Server_Internationalisation).
+    More information on how to get started with translations and beyond
+    is available at
+    [MusicBrainz Server internationalization](https://musicbrainz.org/doc/MusicBrainz_Server/Internationalization).
 
 2.  Change to the po directory
 
         cd po/
 
-3.  Get translations
-
-        tx pull -l {a list of languages you want to pull}
-
-    This will download the .po files for your language(s) of choice to the po/
-    folder with the correct filenames. Languages are written as an ISO language
-    code, optionally followed by an underscore and an ISO country code (e.g. `fr`
-    for French, `fr_CA` for Canadian French).
+3.  Get the latest translations
 
-    Or, if you want to get _all_ translations instead:
+    Weblate is configured to follow the `*.pot` files from the branch `beta`
+    and to push its updated `*.po` files to to the branch `translations`.
```

Comment:
Should we mention that we periodically merge the translations branch into beta/master?  (I'm not sure it's 100% clear that this step is unnecessary unless you want "bleeding edge" translations.)

Reply:
Indeed I assumed that this section was about working with “bleeding edge” translations, that’s why I added “the latest” to the title. I just clarified this point in the commit https://github.com/metabrainz/musicbrainz-server/pull/3024/commits/d08c9ae5c150e5a9b151f28d986e83da98d25f1a.